randovania/games/dread/logic_database/Artaria.txt

```diff
@@ -4952,7 +4952,12 @@ Extra - asset_id: collision_camera_081
   > Event - Screw Attack Water Blob
       Any of the following:
           Wave Beam or Shoot Diffusion Beam
-          Pseudo-Wave Beam (Advanced) and Shoot Beam
+          All of the following:
+              Pseudo-Wave Beam (Advanced) and Shoot Charge Beam
+              Any of the following:
+                  Slide
+                  No Gravity Suit and Enabled Highly Dangerous Logic
```

Comment:
why the slide req?

Reply:
So this is a thing I added for the purpose of making this be non-dangerous. Since there's a lower difficulty version that requires slide, if you have slide, it's not a problem that you pick up gravity suit. Since often, and will probably always, have slide as a starting item, this allows gravity suit to not be a dangerous resource.

Reply:
this makes sense, although charge isn't needed for the pseudo.